src/content/tools/analyzer-performance.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Troubleshoot Analyzer Performance
+description: Help users with analyzer performance issues.
```

Comment:
Consider writing this for a developer wondering why they should read this. Perhaps:

```suggestion
description: >-
  Learn how to diagnose and resolve common performance issues
  with the Dart analysis server.
```

Reply:
This sounds to me like it's equivalent to "Learn how to use the Dart analysis server to diagnose and resolve common performance issues.". Perhaps:

Learn how to diagnose and resolve common Dart analysis server performance issues.